No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'
Traceback (most recent call last):
  File "/home/khalevy/Multilingual-Fig-QA/rag/main.py", line 6, in <module>
    from lib.pipeline import RAGPipeline
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/pipeline.py", line 5, in <module>
    from lib.retrieval import RetrieverBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/retrieval.py", line 15, in <module>
    from lib.preprocess import DatasetBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/preprocess.py", line 48, in <module>
    'char_text_splitter': CharacterTextSplitter(
                          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/character.py", line 16, in __init__
    super().__init__(**kwargs)
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/base.py", line 54, in __init__
    raise ValueError(
ValueError: Got a larger chunk overlap (250) than chunk size (150), should be smaller.
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'
Traceback (most recent call last):
  File "/home/khalevy/Multilingual-Fig-QA/rag/main.py", line 6, in <module>
    from lib.pipeline import RAGPipeline
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/pipeline.py", line 5, in <module>
    from lib.retrieval import RetrieverBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/retrieval.py", line 15, in <module>
    from lib.preprocess import DatasetBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/preprocess.py", line 48, in <module>
    'char_text_splitter': CharacterTextSplitter(
                          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/character.py", line 16, in __init__
    super().__init__(**kwargs)
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/base.py", line 54, in __init__
    raise ValueError(
ValueError: Got a larger chunk overlap (250) than chunk size (150), should be smaller.
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'
Traceback (most recent call last):
  File "/home/khalevy/Multilingual-Fig-QA/rag/main.py", line 6, in <module>
    from lib.pipeline import RAGPipeline
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/pipeline.py", line 5, in <module>
    from lib.retrieval import RetrieverBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/retrieval.py", line 15, in <module>
    from lib.preprocess import DatasetBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/preprocess.py", line 48, in <module>
    'char_text_splitter': CharacterTextSplitter(
                          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/character.py", line 16, in __init__
    super().__init__(**kwargs)
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/base.py", line 54, in __init__
    raise ValueError(
ValueError: Got a larger chunk overlap (250) than chunk size (150), should be smaller.
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'
Traceback (most recent call last):
  File "/home/khalevy/Multilingual-Fig-QA/rag/main.py", line 6, in <module>
    from lib.pipeline import RAGPipeline
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/pipeline.py", line 5, in <module>
    from lib.retrieval import RetrieverBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/retrieval.py", line 15, in <module>
    from lib.preprocess import DatasetBase
  File "/home/khalevy/Multilingual-Fig-QA/rag/lib/preprocess.py", line 48, in <module>
    'char_text_splitter': CharacterTextSplitter(
                          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/character.py", line 16, in __init__
    super().__init__(**kwargs)
  File "/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain_text_splitters/base.py", line 54, in __init__
    raise ValueError(
ValueError: Got a larger chunk overlap (250) than chunk size (150), should be smaller.
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'
/home/khalevy/miniconda3/lib/python3.11/site-packages/langchain/embeddings/__init__.py:29: LangChainDeprecationWarning: Importing embeddings from langchain is deprecated. Importing from langchain will no longer be supported as of langchain==0.2.0. Please import from langchain-community instead:

`from langchain_community.embeddings import HuggingFaceEmbeddings`.

To install langchain-community run `pip install -U langchain-community`.
  warnings.warn(
/home/khalevy/miniconda3/lib/python3.11/site-packages/datasets/load.py:2516: FutureWarning: 'use_auth_token' was deprecated in favor of 'token' in version 2.14.0 and will be removed in 3.0.0.
You can remove this warning by passing 'token=<use_auth_token>' instead.
  warnings.warn(
Repo card metadata block was not found. Setting CardData to empty.
slurmstepd: error: *** JOB 243632 ON shire-1-6 CANCELLED AT 2024-04-21T17:37:58 ***
